ios/FluentUI/Notification/FluentNotification.swift

```diff
@@ -329,6 +351,8 @@ public struct FluentNotification: View, ConfigurableTokenizedControl {
     @State private var innerContentsSize: CGSize = CGSize()
     @State private var attributedMessageSize: CGSize = CGSize()
     @State private var attributedTitleSize: CGSize = CGSize()
+    @available(iOS 15.0, *)
+    @AccessibilityFocusState var isNotificationFocused: Bool
```

Comment:
is it possible to add private on accessibilityfocusstate variable like normal state var?